src/workloads/query/plan_cache/EmptyGroup.yml

```diff
@@ -0,0 +1,248 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test was created to compare using the Classic vs SBE plan caches, for an SBE query,
+  by testing a worst case.
+  
+  The test uses a $group query to ensure the query is SBE-eligible, but uses an empty collection
+  to minimize the execution time.
```

Comment:
[nit] "to minimize the proportion of the query execution time relative to query planning"

(in case the reader doesn't realize "execution" here means specifically query execution)